bears/vcs/git/GitCommitBear.py

```diff
@@ -203,3 +239,85 @@ def check_body(self, body,
             yield Result(self, 'Body of HEAD commit contains too long lines. '
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
+
+    def check_issue(self, body,
+                    body_regex: str=None,
+                    body_close_issue: bool=False,
+                    body_close_issue_full_url: bool=False,
+                    body_close_issue_on_last_line: bool=False):
+        """
+        Check for matching issue related references and URLs.
+
+        :param body:
+            Body of the commit message of HEAD.
+        :param body_regex:
+            If provided, matches the keywords with the regex.
+        :param body_close_issue:
+            Whether to check for the presence of issue close reference
+            within the commit body by retrieving host information from git
+            configuration. GitHub and GitLab support auto closing issues with
+            commit messages. Checks for matching keywords in the commit body.
+            By default, if none of  ``body_close_issue_full_url`` and
+            ``body_close_issue_on_last_line`` are enabled, checks for presence
+            of short references like ``closes #213``, etc. Otherwise behaves
+            according to other chosen flags.
+            More on keywords follows.
+            [GitHub keywords](https://help.github.com/articles/closing-issues-via-commit-messages/)
+            [GitLab keywords](https://docs.gitlab.com/ce/user/project/issues/automatic_issue_closing.html)
+        :param body_close_issue_full_url:
+            Checks the presence of issue close reference with a full URL
+            related to some issue. Works along with ``body_close_issue``.
+        :param body_close_issue_on_last_line:
+            When enabled, checks for issue close reference presence on the
+            last line of the commit body. Works along with
+            ``body_close_issue``.
+        """
+        if body_regex:
+            if not re.search(body_regex, body):
+                yield Result(self, 'No match found for the regular '
+                                   'expression provided: %s' % body_regex)
+            return
```

Comment:
No return, as I pointed out we want the issue check still be running though a regex was selected for the body^^